jac-uk/qt
# setup beta site for QT tests

## What's included?
- Setup beta site for QT tests.
- This site only contains the code required to allow candidates to take their qualifying tests.
- i.e. Vacancy and Applications info is removed from this website, and will remain on the Apply portal instead


## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Check that qualifying tests can be sat, using the new (beta) site
- Check that links to Vacancy and Applications direct to the Apply site.

When testing, developers should checkout the qt-separation branch on the apply repo at the same time as the setup-beta-site branch on online-tests repo and run both sites at the same time. They should find that their browser switches between the two sites seamlessly:
- Apply = http://localhost:8081
- Online Tests = http://localhost:8083

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work
🟠 Medium risk - this does change code that is shared with other areas
🔴 High risk - this includes a lot of changes to shared code

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
